.github/workflows/build-package-docs.yaml

```diff
@@ -0,0 +1,95 @@
+name: Ansible package docs build
+on:
+  workflow_dispatch:
+    inputs:
+      github_fork:
+        description: 'GitHub Fork'
+        required: true
+        default: 'ansible'
+      github_branch:
+        description: 'GitHub Branch'
```

Comment:
It would be useful to document this as a reference or a commitish as it actually doesn't only accept branches, but also tags and commit SHAs.